editor/debugger/debug_adapter/debug_adapter_types.h

```diff
@@ -190,8 +190,8 @@ struct Message {
 struct Scope {
 	String name;
 	String presentationHint;
-	int variablesReference;
-	bool expensive;
+	int variablesReference = 0;
+	bool expensive = false;
```

Comment:
Fixes this type of errors:
![image](https://github.com/godotengine/godot/assets/4701338/0fe2ef91-6b68-4c4d-8904-9782a596e3d7)

CC @rsubtil

Reply:
Good catch, `expensive` ended up never being set in the code, so this was actually causing undefined behavior.

I can confirm it should be `false` as AFAIK Godot still doesn't support partial/pagination behavior to requesting variables from stack dumps, so there's no point in hinting such behavior to DAP when Godot will already have all that information.